cmd/roy/harvest_wikidata.go

```diff
@@ -21,12 +21,16 @@ import (
 	"os"
 
 	"github.com/richardlehane/siegfried/pkg/config"
-	"github.com/ross-spencer/spargo/pkg/spargo"
+	"github.com/ross-spencer/wikiprov/pkg/spargo"
 )
 
 // harvestWikidata will connect to the configured Wikidata query service
 // and save the results of the configured query to disk.
 func harvestWikidata() error {
+
+	const historyLen int = 5
```

Comment:
Users might want more history in Roy's `inspect` function. So also configurable?